src/components/profile/profile.component.js

```diff
@@ -30,6 +33,18 @@ class Profile extends React.Component {
 
   /** Returns the avatar portion of the profile. */
   get avatar() {
+    if (this.props.src) {
```

Comment:
I guess this could be replaced with:
```suggestion
    if (this.hasSrc) {
```